packages/playground/wordpress/src/boot.ts

```diff
@@ -169,6 +177,8 @@ export async function bootWordPress(options: BootOptions) {
 		documentRoot: options.documentRoot || '/wordpress',
 		absoluteUrl: options.siteUrl,
 		rewriteRules: wordPressRewriteRules,
+		getFileNotFoundAction:
+			options.getFileNotFoundAction ?? getFileNotFoundActionForWordPress,
```

Comment:
This WP-compatible default should allow PHPRequestHandler to work well for any users of `bootWordPress()`. And the worker thread can override the callback to add handling for known remote assets.